src/components/pages/Dashboard/DashboardContainer.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import RenderDashboardPage from './RenderDashboardPage';
 import TitleComponent from '../../common/Title';
-
+import ServiceMap from './ServiceMap';
```

Comment:
Very good job at adding the ServiceMap to the DashboardContainer! This will for sure help users to understand services that have been provided in the area